lib/rspecq/queue.rb

```diff
@@ -241,6 +242,22 @@ def flaky_jobs
       requeued - @redis.hkeys(key_failures)
     end
 
+    def fail_fast_threshold
+      @fail_fast || -1
+    end
+
+    # Returns true if the number of failed tests, has surpassed the threshold
+    # to render the run unsuccessful. This is also used internally to block any
+    # new jobs being enqueued
+    def suite_failed?
```

Comment:
Since this is specific to the fail-fast option, perhaps we should rename it to something less generic, in order to avoid confusion. For example, you can't use this method without `fail-fast` set.

How about `fail_fast_triggered?`.

Reply:
Actually, this should be named consistently with the `build_successful?` method. So for example `build_failed_fast?` is a good name?